src/makeCalculator.js

```diff
@@ -37,7 +37,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let output = 0;
+
+  return {
+    add(x) {
+      output += x;
+
+      return this;
+    },
+
+    subtract(x) {
+      output -= x;
+
+      return this;
+    },
+
+    multiply(x) {
+      output *= x;
+
+      return this;
+    },
+
+    divide(x) {
+      output /= x;
+
+      return this;
+    },
```

Comment:
You need to add a check for 0, because by the rules of mathematics, you cannot divide by 0